app/editor/src/features/admin/users/UserList.tsx

```diff
@@ -11,30 +14,62 @@ import { UserFilter } from './UserFilter';
 
 export const UserList: React.FC = () => {
   const navigate = useNavigate();
-  const [{ users }, api] = useUsers();
-  const [{ requests }] = useApp();
-
+  const [{ users, userFilter }, { findUsers, storeFilter }] = useUsers();
+  const [{ requests }, { isUserReady }] = useApp();
   const [page, setPage] = React.useState(
     new Page(users.page - 1, users.quantity, users.items, users.total),
   );
 
+  const handleChangeSort = React.useCallback(
+    (sortBy: SortingRule<IUserModel>[]) => {
+      const sorts = sortBy.map((sb) => ({ id: sb.id, desc: sb.desc }));
+      const same = sorts.every(
+        (val, i) => val.id === userFilter.sort[i]?.id && val.desc === userFilter.sort[i]?.desc,
+      );
+      if (!same) {
+        storeFilter({ ...userFilter, sort: sorts });
+      }
+    },
+    [storeFilter, userFilter],
+  );
+
+  const fetch = React.useCallback(
+    async (filter: IUserFilter) => {
+      try {
+        const data = await findUsers(makeUserFilter(filter));
```

Comment:
`IUserFilter` is already the correct shape for the API.  Why do you need to now convert it?

Reply:
This was to make sure sort was working correctly (getting the appropriate column etc)